polaris.shopify.com/src/components/Button/Button.tsx

```diff
@@ -1,13 +1,16 @@
-import { HTMLProps } from "react";
+import Link, { LinkProps } from "next/link";
+import { HTMLProps, PropsWithChildren } from "react";
 import styles from "./Button.module.scss";
 
-interface Props extends HTMLProps<HTMLButtonElement> {
+interface Props {
   small?: boolean;
   pill?: boolean;
-  children: React.ReactNode;
 }
 
-function Button({ small, pill, children, ...rest }: Props) {
+interface NativeButtonProps extends Props, HTMLProps<HTMLButtonElement> {}
```

Comment:
```suggestion
interface ButtonProps extends Props, HTMLProps<HTMLButtonElement> {}
```

Light suggestion to keep prop interfaces consistent with the associated component.

Reply:
You mean the name of the interface? (Definitely agree!)